test/services/school/index.test.js

```diff
@@ -283,29 +266,30 @@ describe('school service', () => {
 			expect(result.features).to.include('rocketChat');
 		});
 
-		it('team creation by students should be updated according to environment setting'
-			+ ' without admin setting', async () => {
-			const school = await testObjects.createTestSchool({});
+		it(
+			'team creation by students should be updated according to environment setting' + ' without admin setting',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)